ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-answer/activityAnswer.component.ts

```diff
@@ -1,86 +1,107 @@
-                                           [class]="'composite-answer-' + block.stableId"
-                                           [block]="block"
-                                           [readonly]="readonly"
-                                           [validationRequested]="validationRequested"
-                                           (valueChanged)="onChange($event)"
-                                           (componentBusy)="componentBusy.next($event)">
-            </ddp-activity-composite-answer>
-            <ddp-activity-agreement-answer *ngIf="isAgreementQuestion(block)"
-                                           [block]="block"
-                                           [readonly]="readonly"
-                                           (valueChanged)="onChange($event)">
-            </ddp-activity-agreement-answer>
-            <ddp-activity-file-answer *ngIf="isFileQuestion(block)"
-                                      [class]="'file-answer-' + block.stableId"
-                                      [block]="block"
-                                      [readonly]="readonly"
-                                      [studyGuid]="studyGuid"
-                                      [activityGuid]="activityGuid"
-                                      (valueChanged)="onChange($event)"
-                                      (componentBusy)="componentBusy.next($event)">
-            </ddp-activity-file-answer>
-            <ddp-activity-matrix-answer *ngIf="isMatrixQuestion(block)"
-                                        [class]="'matrix-answer-' + block.stableId"
-                                        [block]="block"
-                                        [readonly]="readonly"
-                                        (valueChanged)="onChange($event)">
-            </ddp-activity-matrix-answer>
-            <ddp-activity-instance-select-answer *ngIf="isActivityInstanceSelectQuestion(block)"
-                                                 [class]="'activity-instance-select-answer-' + block.stableId"
-                                                 [block]="block"
-                                                 [readonly]="readonly"
-                                                 (valueChanged)="onChange($event)"
-                                                 (componentBusy)="componentBusy.next($event)"
-            ></ddp-activity-instance-select-answer>
-            <span *ngIf="block.additionalInfoFooter"
-                  [innerHTML]="block.additionalInfoFooter"
-                  class="ddp-activity-answer__info-footer">
-      </span>
-        </ng-container>`,
-    styleUrls: ['activityAnswer.component.scss']
+    template: ` <ng-container *ngIf="block.shown">
+        <ddp-activity-boolean-answer
+            *ngIf="isBooleanQuestion(block)"
+            [class]="'boolean-answer-' + block.stableId"
+            [block]="block"
+            [readonly]="readonly"
+            (valueChanged)="onChange($event)"
```

Comment:
Are all the changes to this component's template formatting? If so can you indicate it in a comment. Don't think the reformatting is unreasonable but I am expending time and focus trying to figure out if this is indeed the case.